Bottom: bddd14ff75da41281e71f7927586250e01b691b2
Top:    29f652f1245401a0a9d8cf0755d0b8525993bd2e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-01 00:21:45 +0100

Refresh of polygon1-c-temporarily-disable

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 78ba618..5a91370 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -600,6 +600,7 @@ adjust_tree (rtree_t * tree, struct seg *s)
   return 0;
 }
 
+#if 0
 /*
  * seg_in_region()
  * (C) 2006, harry eaton
@@ -621,6 +622,7 @@ seg_in_region (const BoxType * b, void *cl)
     return 0;
   return 1;			/* might intersect */
 }
+#endif
 
 /* Prepend a deferred node-insersion task to a list */
 static insert_node_task *
@@ -844,7 +846,7 @@ contour_bounds_touch (const BoxType * b, void *cl)
       info.tree = rtree_over->tree;
       if (info.tree)
 	if (UNLIKELY (r_search (info.tree, &info.s->box,
-				seg_in_region, seg_in_seg, &info)))
+				NULL/* seg_in_region */, seg_in_seg, &info)))
 	  assert (0); /* XXX: Memory allocation failure */
     }
   while ((av = av->next) != &looping_over->head);
